buildah/buildah.yaml

```diff
@@ -6,6 +6,7 @@ spec:
   parameters:
   - name: BUILDER_IMAGE
     description: The location of the buildah builder image.
+    default: docker.io/vdemeester/buildah-builder:latest
```

Comment:
@ImJasonH @vdemeester do we want to have defaults that exist or rather just expect users to build their own? Having a default like this would better be rebuilt regularly and would bear maintaining.

Reply:
@vbatts right, so… I wasn't really sure about that. I think, for the use, it's better to have a default one that is up-to-date (automated somehow)… but maybe we ain't ready yet to do that :angel: :stuck_out_tongue_closed_eyes: